pkg/config/config.go

```diff
@@ -137,9 +137,12 @@ func init() {
 	Datadog.SetDefault("expvar_port", "5000")
 	// Agent GUI access port
 	Datadog.SetDefault("GUI_port", "-1")
-	// Proess Agent
+	// Process Agent
 	Datadog.SetDefault("process_agent_enabled", true) // this is to support the transition to the new config file
 
+	// Log Agent
+	Datadog.SetDefault("log_agent_enabled", true)
```

Comment:
since you opened this PR, the config parameter has been added in a separate PR (and is different), see https://github.com/DataDog/datadog-agent/pull/844